tests/e2e/creation_test.go

```diff
@@ -444,6 +451,48 @@ func (tc *testContext) testDefaultCertsAvailable() error {
 	return nil
 }
 
+func (tc *testContext) testTrustedCABundle() error {
```

Comment:
not sure i understand the logic here:
if we are making e2e test (not unit-test) , what we should test is:
1. set managementStateChangeTrustedCA := true

- check if Configmap is created in tc.testDSCI.Spec.ApplicationsNamespace
- check content of CADataFieldName == tc.testDSCI.Spec.TrustedCABundle.CustomCABundle
- check content of CADataFieldName has newline as ending (this is new from current code base)

2. set managementStateChangeTrustedCA := false
- check no Configmap  in tc.testDSCI.Spec.ApplicationsNamespace

Reply:
ok, seems confusion for the above comments:
`set managementStateChangeTrustedCA := true`
is not meant to be set in the testcase, this is only to illustrate the case when set to `Managed` VS `Removed`
what need to be done is `tc.testDSCI.Spec.TrustedCABundle.ManagementState: Managed`